### ISSUE TITLE
can't specify CSV field separator

### ISSUE THREAD

**Issue by helix84** [xo/usql#96]
```
usql --set SHOW_HOST_INFORMATION=false mysql:// -c "SELECT * FROM items LIMIT 1;" -C`
# this works (outputs pipe-separated CSV)

usql --set SHOW_HOST_INFORMATION=false mysql:// -c "SELECT * FROM items LIMIT 1;" -C -F,
usql --set SHOW_HOST_INFORMATION=false mysql:// -c "SELECT * FROM items LIMIT 1;" -C -F ,
usql --set SHOW_HOST_INFORMATION=false mysql:// -c "SELECT * FROM items LIMIT 1;" -C --field-separator=","
usql --set SHOW_HOST_INFORMATION=false mysql:// -c "SELECT * FROM items LIMIT 1;" -C --field-separator=,
# all of the above return the following error:
usql: error: --csv must be passed a value, try --help
```

**Comment by kenshaw** [comment]
Thanks for pointing this out. I'll make a point of fixing it in the next release.

**Comment by kenshaw** [comment]
This will be fixed in next release.

**Comment by jgoux** [comment]
Hello,
is there any ETA for the next release?

**Comment by shdf2** [comment]
> Hello,
> is there any ETA for the next release?

+1
Any ETA for this please ? This was reported 7 months ago...

Thank you

**Comment by calebeaires** [comment]
Same here, any news about it?

**Comment by Jeremy-Aktify** [comment]
I agree, this also doesn't work for me.

**Comment by nineinchnick** [comment]
Looks like this is already fixed in master, in eaac671fc4ae2c713534434cd2e7884e872baa3e, just not released yet.
